applications/paraview/client_server_setup.md

```diff
@@ -7,18 +7,20 @@ This guide assumes that you already have a copy of the ParaView software install
 For Windows users, it is additionally necessary to install the PuTTY software to complete the steps carried out in the terminal.  You can [download it here](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html).
 
 ### 1. Reserve Compute Nodes
-The first step is to reserve the computational resources on Eagle that will be running the ParaView server.  This requires using the Slurm `srun` directive and specifying an allocation name and time limit for the reservation.
+The first step is to reserve the computational resources on Eagle that will be running the ParaView server. 
+
+This requires using the Slurm `salloc` directive and specifying an allocation name and time limit for the reservation. (This is one of the rare times where `salloc` is used instead of srun.)
```

Comment:
Not sure that salloc is a rare occurrence. This may lead to confusion...

Maybe we explain why we use salloc here and/or point to information on this subject?

Reply:
see latest commit that expands on this section quite a bit.